docker/dev/ubuntu/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Develop Orion Context Broker Ubuntu-Latest
+FROM ubuntu:latest
+
+MAINTAINER FIWARE Orion Context Broker Team. Telefónica I+D
+
+RUN \
+    apt-get update && \
```

Comment:
This script looks a bit obscure for people not used to the Orion build process. Maybe introducing comments in the proper places could help, e.g:

```
# Installing Orion build dependencies
...
# Building mongo driver
...
```